### DISCUSSION TITLE
docs/web/senior-hot-reload

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#89] [General]
# docs/web/senior-hot-reload

在使用GoFrame框架时启用和管理平滑重启特性。通过该功能，WebServer在更新版本或者重启时，无需中断已有请求，从而提高服务稳定性和用户体验。支持在*nix系统下的平滑重启和Windows系统下的完整重启，并提供Web和命令行的管理方法。还展示了具体的使用示例，包括基本使用、HTTPS支持和多服务多端口场景的平滑管理。

https://goframe.org/docs/web/senior-hot-reload

**Comment by korialuo** [comment]
请问：1. g.Server("server1").Shutdown() 方法可以结束另一个进程吗？

**Reply by korialuo** [reply]
2. 最新版本2.8.1中, g.Server() 没有了Restart() 方法。
